presto-testing/src/main/java/io/prestosql/testing/AbstractTestQueryFramework.java

```diff
@@ -83,14 +86,15 @@
     private QueryRunner queryRunner;
     private H2QueryRunner h2QueryRunner;
     private SqlParser sqlParser;
+    private final Closer afterClassCloser = Closer.create();
```

Comment:
nit: use `Closer` from `starburst-presto`

Reply:
Let me handle that in separate pr.